### ISSUE TITLE
remove Author class from ProvONE model

### ISSUE THREAD

**Issue by mbjones** [DataONEorg/sem-prov-design#80]
Currently the `Author` class is in ProvONE as a subclass of `prov:Agent`, and was intended to be used in `Program wasAtrributedTo Author` triples.  However, `Author` is a static piece of metadata about about the program that in DataONE will not change across invocations, and would overlap with other metadata about the `Program` which we obtain from the DataONE registered metadata document for a `Program`, just as we do for `Data` objects in DataONE.  This metadata should be provided by the submitter of the Program to DataONE, and not by the person running the program.  When displaying a `Program` in DataONE user interfaces, the `Creator` from the metadata document associated with the `Program` should be attributed as the creator of the work. DataONE should ignore any isAttributedTo statements from the provenance trace because these could conflict with the original `Program` author's intentions.  If the `Program` was modified and was really a different program from the original, then a new version with a new identifier should be uploaded to DataONE, it's `Creator` field should be set in the metadata for the `Program`, and then its new identifier can be referenced in any provenance traces as needed.  This also allows other critical metadata about the program to be provided, such as its `License`, `Title`, `Description`, etc.  See the [codemeta](https://github.com/mbjones/codemeta) project which is crosswalking software metadata standards to come up with a canonical set of information for registering software in repositories.

A good rule of thumb for which metadata properties should be part of the ProvONE spec would be that those properties which might change from one `Execution` to another would be in ProvONE, whereas static properties of `Entity` subclasses (like the title, author, and license of a `Data`, `Visualization`, or `Program` object) are in separate metadata standards. To close this task, we simply need to:

- [ ] remove Author and any connected properties from the ProvONE OWL files, diagrams, and documentation.

**Comment by csjx** [comment]
Yes, we recently introduced `Author` in the ProvONE model, whereas before only `User` was present.  Removing it should be inconsequential.  The `User`class will remain.
